ui/src/pages/insightDetail/components/eventAggregator/styles.module.less

```diff
@@ -1,10 +1,7 @@
 .events_container {
   display: flex;
   flex-direction: column;
-  background: #fff;
-  border-radius: 8px;
-  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
-  padding: 24px;
+  padding: 0 24px 24px 24px;
   position: relative;
```

Comment:
The padding has been changed to `0 24px 24px 24px`. This removes the top padding while keeping the bottom padding. Please confirm if this change affects the layout as expected.